### PR TITLE
Added script to package repo into zip file

### PR DESCRIPTION
**Pull request by ErinLMoore**
This script makes sure that you are on an up-to-date master branch, and then creates the .zip file to be uploaded to s3. I attempted to make the functionality similar to what we do to check the git branch in root-infrastructure.

### CODE REVIEW

File: package-to-zip.sh [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  exit 0
+fi
+
+
+CURRENT_BRANCH=`git rev-parse --abbrev-ref HEAD`
+
+RED='\033[0;31m'
+NC='\033[0m'
+
+if [ $CURRENT_BRANCH != "script-to-package-elm" ]; then
```
Comment (atareshawty) [inline package-to-zip.sh]: debugging statement?

Comment (ErinLMoore) [reply package-to-zip.sh]: Oh geez, oops. Thanks for the catch.

**Review by atareshawty — COMMENTED** [review]
I'm not quite sure fast-forwarding is desired behavior when running a script called `package-to-zip.sh`. IMO this should print out a warning if you're on master and it's behind but lets you package up whatever ref you want. What if I want to test my changes in a testing lambda?

**Comment by coreycarvalho** [comment]
> IMO this should print out a warning if you're on master and it's behind but lets you package up whatever ref you want.

I also prefer this recommendation.

Additionally, I'd prefer to see this behavior in CI. It might be worth evaluating whether or not we want to treat this codebase as a first class citizen since it is running in production (probably not as part of this work).

**Review by nijave — CHANGES_REQUESTED** [review]
How about skipping checking against master, but still check that the branch is up to date with the remote & include the branch name in the output file. That would allow building on branches using the same script but safeguarded against building against outdated stuff?

File: README.md [hunk 2]
```diff
@@ -20,10 +20,15 @@ Install virtualenv:
...
+Make sure your current branch is clean and up-to-date, and run the shell script `package-to-zip.sh`. You can pass an optional output filename to it as an argument.
+This script will create a .zip file in the directory you run it in.
+
+Alternately, here are the steps to create the .zip file manually:
+
 1. CD to the folder that contains main.py.
 1. Run the following command:
-`pip install -r requirements.txt -t `pwd``
-1. Compress the contents of folder (not the folder).
+`pip install -r requirements.txt -t .`
```
Comment (atareshawty) [inline README.md:30]: If we plan on pushing these changes up stream, I'm not sure we want to change these instructions

**Review by atareshawty — APPROVED** [review]

**Review by coreycarvalho — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (ErinLMoore) [reply README.md:30]: I wasn't going to push this PR upstream. It won't really be necessary if we're not maintaining this repo.

**Pull request closed without merging**